android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlBroadcastReceiver.java

```diff
@@ -254,7 +269,7 @@ private static void startRouterService(Context context, ComponentName componentN
         }
     }
 
-    private boolean wakeUpRouterService(final Context context, final boolean ping, final boolean altTransportWake, final BluetoothDevice device) {
+    private boolean wakeUpRouterService(final Context context, final boolean ping, final boolean altTransportWake, final BluetoothDevice device, final VehicleType vehicleType) {
```

Comment:
@kostyaBoss On line 272 I would remove `final VehicleType vehicleType` from the method signature.

Reply:
@santhanamk, can't do this, referencing this comment https://github.com/smartdevicelink/sdl_java_suite/pull/1604#discussion_r563845919